account_bank_statement_import_online_qonto/models/online_bank_statement_provider_qonto.py

```diff
@@ -62,6 +62,11 @@ def _qonto_obtain_transactions(self, slug, date_since, date_until):
         self.ensure_one()
         url = QONTO_ENDPOINT + "/transactions"
         params = {"slug": slug, "iban": self.account_number}
+        # Avoid errors if year is different (set last day of the year)
```

Comment:
Include the error you have if not:
```suggestion
        # Avoid this error if the end year is different: "The datetime in 
        # settled_at_to param isn't well formatted (ISO 8601) or year is out
        # of range". We set the last day of the year in such case.
```